pkg/scheduler/framework/interface.go

```diff
@@ -184,6 +190,16 @@ type Framework interface {
 	// cluster state to make the pod potentially schedulable in a future scheduling cycle.
 	RunPostFilterPlugins(ctx context.Context, state fwk.CycleState, pod *v1.Pod, filteredNodeStatusMap fwk.NodeToStatusReader) (*fwk.PostFilterResult, *fwk.Status)
 
+	// Get a "node hint" for a given pod. If the function returns anything but the empty string, then
+	// this node can be evaluated
+	RunNodeHint(ctx context.Context, pod *v1.Pod, state fwk.CycleState, nodeInfos fwk.NodeInfoLister) string
+
+	// Store the results after we have sorted and filtered nodes.
+	StoreScheduleResults(ctx context.Context, chosenNode string, otherNodes SortedScoredNodes)
+
+	// Called when a pod successfully schedules.
+	RunPostScore(ctx context.Context, thisFramework bool, success bool)
```

Comment:
nit: Please rethink about the function naming, or add more description to the comments at the very least. 
- What is "PostScore"?
- What is "NodeHint"?